op_builder/xpu/builder.py

```diff
@@ -6,9 +6,11 @@
 import os
 import time
 import importlib
+import shutil
+from pathlib import Path
+import subprocess
 from deepspeed.ops.op_builder.builder import OpBuilder, TORCH_MAJOR, TORCH_MINOR
 
-
 class SYCLOpBuilder(OpBuilder):
```

Comment:
shouldn't there be two kinds of builder, SYCLOpBuilder and SYCLAutoOpBuilder, and SYCLAutoOpBuilder works for ops with syclomatic?

Reply:
> shouldn't there be two kinds of builder, SYCLOpBuilder and SYCLAutoOpBuilder, and SYCLAutoOpBuilder works for ops with syclomatic?

add SYCLAutoOpBuilder below.